modules/nf-core/lai/main.nf

```diff
@@ -0,0 +1,71 @@
+process LAI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ltr_retriever:2.9.0--hdfd78af_2':
+        'biocontainers/ltr_retriever:2.9.0--hdfd78af_2' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path pass_list
+    path annotation_out
+    path monoploid_seqs
+
+    output:
+    tuple val(meta), path("*.LAI.log")  , emit: log
+    tuple val(meta), path("*.LAI.out")  , emit: lai_out     , optional: true
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args            = task.ext.args     ?: ''
+    def prefix          = task.ext.prefix   ?: "${meta.id}"
+    def monoploid_param = monoploid_seqs    ? "-mono $monoploid_seqs"                       : ''
+    def lai_output_name = monoploid_seqs    ? "${annotation_out}.${monoploid_seqs}.out.LAI" : "${annotation_out}.LAI"
```

Comment:
When `monoploid_seqs` is `[]` this name is going to be `${annotation_out}..out.LAI` (i.e. with the double `.`). Is that really the case when `monoploid_seqs` is not given, otherwise the `mv` will fail?

Reply:
Good point. I need to add a test to verify this functionality.

Reply:
Now I am testing this behaviour here https://github.com/nf-core/modules/blob/1e81d01bac78b08903346dac46bfa2feda69dfae/modules/nf-core/ltrretriever/lai/tests/main.nf.test#L104

and here https://github.com/nf-core/modules/blob/1e81d01bac78b08903346dac46bfa2feda69dfae/modules/nf-core/ltrretriever/lai/tests/main.nf.test#L135

Thank you @mahesh-panchal